src/main/java/seatsio/events/Events.java

```diff
@@ -304,6 +304,13 @@ public ChangeObjectStatusResult changeObjectStatus(String eventKey, List<?> obje
         return changeObjectStatus(singletonList(eventKey), objects, status, holdToken, orderId, keepExtraData, ignoreChannels, channelKeys, allowedPreviousStatuses, rejectedPreviousStatuses);
     }
 
+    public ChangeObjectStatusResult changeObjectStatus(List<String> eventKeys, List<?> objects, String status) {
```

Comment:
Do we need a new method here, or can we use one of the existing methods that do the API call? The less places that do the actual call the better imho (less chance for bugs).

Reply:
Agreed, and updated